examples/fpga/artya7/util/vivado_hook_route_design_post.tcl

```diff
@@ -0,0 +1,27 @@
+open_project ../../lowrisc_ibex_top_artya7_0.1.xpr
+set saif_name "detailed_power.saif"
+
+regexp -- {Vivado v([0-9]{4})\.[0-9]} [version] -> year
+create_run impl_2 -parent_run synth_1 -flow "Vivado Implementation $year"
+set_property STEPS.OPT_DESIGN.IS_ENABLED false [get_runs impl_2]
+launch_runs impl_2
+wait_on_run impl_2
```

Comment:
Can you please add comments here explaining why you need to run another implementation and why you need to disable optimization of the design for this?

Reply:
Also, why do you need to extract the year of the Vivado release?

Reply:
I tried to leave a detailed explanation, can you please check it? Thanks

Reply:
Perfect, this is very useful. Thanks!

Reply:
This got resolved as well. The second implementation run without optimization is no longer needed.